File: app/src/main/java/com/erebor/tomkins/pos/tools/SecurityEncryption.java
package com.erebor.tomkins.pos.tools;

public interface SecurityEncryption {
    String md5(String text);
    String aesEncrypt(String textToEncrypt) throws Exception;
    String aesDecrypt(String textToDencrypt) throws Exception;
    String sh1(String text) throws Exception;
}

File: app/src/main/java/com/erebor/tomkins/pos/tools/SecurityEncryption.kt
package com.erebor.tomkins.pos.tools

interface SecurityEncryption {
    fun md5(text: String): String
    @Throws(Exception::class)
    fun aesEncrypt(textToEncrypt: String): String

    @Throws(Exception::class)
    fun aesDecrypt(textToDencrypt: String): String

    @Throws(Exception::class)
    fun sh1(text: String): String
}
